ui/org.eclipse.pde.ui/plugin.xml

```diff
@@ -2496,4 +2496,47 @@
                contentType="org.eclipse.pde.bndInstructions">
          </presentationReconciler>
       </extension>
+         <extension
+         point="org.eclipse.pde.ui.targetLocationProvisioners">
+      <locationProvider
+            class="org.eclipse.pde.internal.ui.bndtools.RepositoryTargetLocationWizard"
+            icon="platform:/plugin/org.eclipse.pde.ui/icons/bndtools/database.png"
+            id="bndtools.RepositoryTargetLocationProvisioner"
+            name="Bnd Repository">
+         <description>
+            Download plug-ins from a repository defined in a Bnd workspace
+         </description>
+      </locationProvider>
+   </extension>
+   <extension
+         point="org.eclipse.pde.core.targetLocations"
```

Comment:
This and below contribution show errors in the IDE:

```
Description	Resource	Path	Location	Type
Attribute 'canUpdate' of element 'targetLocation' must be defined.	plugin.xml	/org.eclipse.pde.ui	line 2515	Plug-in Problem
Attribute 'canUpdate' of element 'targetLocation' must be defined.	plugin.xml	/org.eclipse.pde.ui	line 2537	Plug-in Problem
```

Please fix either extension point or contribution here, not sure what was intended.